src/GasStation/ArbitrumInfuraGasStation.ts

```diff
@@ -0,0 +1,33 @@
+import axios from 'axios';
+
+import { GasStation } from './GasStation';
+
+async function fetchInfura(apiKey: string) {
+  const response = await axios({
+    url: `https://arbitrum.infura.io/v3/${apiKey}`,
+    method: 'POST',
+    headers: { accept: 'application/json', 'content-type': 'application/json' },
+    data: { id: 1, jsonrpc: '2.0', method: 'eth_maxPriorityFeePerGas' },
```

Comment:
method is still named `eth_maxPriorityFeePerGas` for arbitrum https://docs.infura.io/networks/arbitrum/json-rpc-methods/eth_maxpriorityfeepergas